File: ScaleformUI_Csharp/PauseMenu/Tabs/PlayerListTab.cs
using ScaleformUI.LobbyMenu;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ScaleformUI.PauseMenu
{
    public class PlayerListTab : BaseTab
    {
        private bool _focused;
        private int focus = 0;

        public override bool Focused
        {
            get { return _focused; }
            set
            {
                _focused = value;
            }
        }
        public int Focus
        {
            get => focus;
            set
            {
                focus = value;
                if (Parent != null && Parent.Visible)
                {
                    Parent._pause._pause.CallFunction("SET_PLAYERS_TAB_FOCUS", Parent.Tabs.IndexOf(this), focus);
                }
            }
        }
        public SettingsListColumn SettingsColumn { get; private set; }
        public PlayerListColumn PlayersColumn { get; private set; }
        public PlayerListTab(string name) : base(name)
        {
            _type = 2;
            SettingsColumn = new("", HudColor.NONE);
            PlayersColumn = new("", HudColor.NONE);
        }
    }
}

File: ScaleformUI_Lua/src/PauseMenu/Tabs/PlayerListTab.lua
PlayerListTab = setmetatable({}, PlayerListTab)
PlayerListTab.__index = PlayerListTab
PlayerListTab.__call = function()
    return "BaseTab", "PlayerListTab"
end

function PlayerListTab.New(name)
    local data = {
        Base = BaseTab.New(name or "", 2),
        LeftItemList = {},
        Label = name or "",
        TextTitle = "",
        SettingsColumn = SettingsListColumn.New("", -1),
        PlayersColumn = PlayerListColumn.New("", -1),
        Index = 0,
        Focused = false,
        _focus = 0,
    }
    return setmetatable(data, PlayerListTab)
end

function PlayerListTab:Focus(_f)
    if _f == nil then 
        return self._focus
    else
        self._focus = _f
        if self.Base.Parent ~= nil and self.Base.Parent:Visible() then
            ScaleformUI.Scaleforms._pauseMenu._pause:CallFunction("SET_PLAYERS_TAB_FOCUS", false, IndexOf(self.Base.Parent.Tabs, self) - 1, self._focus)
        end
    end
end